examples/sveltekit/src/routes/(app)/+page.server.ts

```diff
@@ -99,7 +99,7 @@ export const actions = {
 			}
 		});
 
-		if (!error) {
```

Comment:
🥲

Reply:
Sorry that this unrelated change got included.

Reply:
No no, it's not about change being unrelated. It's the fact that it wasn't caught before and happily lived there :D

Reply:
Yeah it probably wouldn't have been caught since the examples here are only used by those who work on this library and not everyone uses SvelteKit. It's generally the Next.js version that gets checked the most.